plugins/woocommerce/client/blocks/tests/e2e/tests/mini-cart/mini-cart-block.merchant.classic_theme.spec.ts

```diff
@@ -47,25 +44,14 @@ test.describe( 'Merchant → Mini Cart', () => {
 			await expect(
 				await editor.getBlockByName( blockData.slug )
 			).toBeVisible();
-		} );
-		test( 'can only be inserted once', async ( {
-			page,
-			admin,
-			editor,
-		} ) => {
-			await admin.visitWidgetEditor();
+
 			await editor.openGlobalBlockInserter();
 
 			await editor.page
 				.getByRole( 'searchbox', { name: 'Search' } )
 				.fill( blockData.slug );
 
-			const miniCartButton = page.getByRole( 'option', {
-				name: blockData.name,
-				exact: true,
-			} );
-
-			await expect( miniCartButton ).toBeHidden();
+			await expect( miniCartButton ).toBeDisabled();
```

Comment:
Updated because the button is not hidden immediately after inserting, but it does get `aria-disabled="true"` so it can't be inserted.